### DISCUSSION TITLE
How to collapse vertical tab bar with a shortcut?

### DISCUSSION THREAD

**Discussion by BarbzYHOOL** [Floorp-Projects/Floorp#556] [Feature Request]
Hello, in 1.14, I pressed Ctrl+B to hide the sidebar containing the vertical tabs, but now it displays the bookmark sidebar instead
SO how do I hide the vertical tabs bar easily? 
The setting "Collapse Vertical Tab Bar" in about:preferences is not the same feature

**Comment by gunir** [comment]
Yes you can using Custom Actions feature:

Example, collapsing Tab Toolbar: https://i.imgur.com/bTaefgt.gif

First, go to Customizable shortcuts and change Custom Action 1 hotkey to `F1` and Restart.

Go to `about:config` and change value of `floorp.custom.shortcutkeysAndActions.customAction1` 
to
`var toggletarget = 'TabsToolbar'; if (document.getElementById(toggletarget).style.display == 'none') { document.getElementById(toggletarget).style.display = 'unset'; } else { document.getElementById(toggletarget).style.display = 'none' }`

For Vertical Tab, you might need to change something to make it work, mostly CSS so you can easily figure out.

**Comment by lmcr666** [comment]
@gunir hey, so i have whatsapp on a side panel, the first one actually. i wanted to get a keybind to toggle it and found customactions, i understood everything u did in your reply but i couldnt find any way to know the ElementId for the panel, it there a way that you could tell me?
thank you in advance
